lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
When I talked about RTLD_GLOBAL it was in the context of symbol resolution, and that dlopen(0, RTLD_GLOBAL) provides a -special- handle that will search all loaded images for symbols. As far as I know Windows doesn't have such a mechanism.

m_ExeMM->findSymbol calls RTDyldMemoryManager::getSymbolAddressInProcess which does a lot of other lookup. I'm not sure erasing that call on Windows is the best thing to do (at least right now)